src/show/components/index.js

```diff
@@ -2,5 +2,6 @@ import Split from "./Split";
 import Navbar from "./Navbar";
 import TextField from "./TextField";
 import Button from "./Button";
+import Region from "./Region";
```

Comment:
maintain a tree like structure, it will more readable 
```
import Button from "./Button";
import Region from "./Region";
import Navbar from "./Navbar";
import TextField from "./TextField";
```

Reply:
done